src/main/scala/dynamodb/package.scala

```diff
@@ -263,9 +263,9 @@ package object dynamodb {
   implicit val attributeValueToByteArraySet = (x: AttributeValue) => catchAndRethrowConversion { x.getBS.asScala.map(_.array).toSet }
 
   /** string AttributeValue to Boolean */
-  implicit val attributeValueToBoolean = (x: AttributeValue) => catchAndRethrowConversion { x.getS.toBoolean }
+  implicit val attributeValueToBoolean = (x: AttributeValue) => catchAndRethrowConversion { x.getBOOL }
   /** string set AttributeValue to Set[Boolean] */
-  implicit val attributeValueToBooleanSet = (x: AttributeValue) => catchAndRethrowConversion { x.getSS.asScala.map(_.toBoolean).toSet }
+  implicit val attributeValueToBooleanSet = (x: AttributeValue) => catchAndRethrowConversion { x.getL.asScala.map(_.getBOOL).toSet }
```

Comment:
I think we need to have alternative in case of failure for all the `.getSS` because `.getSS` can also be `.getL + .getS`.
I had that problem when reading something written by a Python client